### DISCUSSION TITLE
Github 3rd party sign in - instant access revocation

### DISCUSSION THREAD

**Discussion by BeMoreDog** [ory/kratos#2290] [Q&A] [answered]
Hi 👋🏻 We’ve successfully enabled sign in with Github.  

In our testing we have revoked access for our app (from our github settings page) but we are still able to continue using our app as the session is still valid and hasn’t expired.  

Is there a recommended way to check whether social sign in access has been revoked so that we can invalidate the session?

**Comment by vinckr** [comment, marked as answer]
Hello @BeMoreDog 
I think this is the same feature as requested in https://github.com/ory/kratos/issues/1912

There is a draft PR in the works if you want to take a look.

There is probably a workaround but I would recommend waiting for the PR to be merged and released.

**Reply by BeMoreDog** [reply]
Great thanks, I'll keep an eye on it
